src/node/rpc/frontend.h

```diff
@@ -252,6 +253,8 @@ namespace ccf
         }
       }
 
+      endpoint->metrics.calls++;
```

Comment:
Is there a reason that the `404`s and `405`s above aren't counted in any of the metrics?

Reply:
Sorry, I don't understand what you mean by that.

Reply:
If you mean the failure to dispatch, I don't know what endpoint we would attribute those to. Perhaps they ought to be their own category?

Reply:
Ah of course, any errors before this aren't attributable to a specific endpoint. Fair enough! I'd be tempted to thrown in a global "dispatch failures" counter, but ideally that'd be incremented from everywhere that returns a pre-dispatch error, which is awkward. So I think ignoring them for now is fine!